platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/WorkflowI18n.java

```diff
@@ -57,6 +57,8 @@
 		.de("Delegierung");
 	I18n0 DRAFT = new I18n0("Draft")//
 		.de("Entwurf");
+	I18n0 EMAIL_NOTIFICATIONS_FOR_NEW_TASKS = new I18n0("Email notifications for new tasks")//
```

Comment:
```suggestion
	I18n0 EMAIL_NOTIFICATIONS_FOR_NEW_TASKS = new I18n0("Email Notifications for New Tasks")//
```
I think we should use title-case here.